ansible-scylla-node/tasks/upgrade/upgrade_debian9.yml

```diff
@@ -0,0 +1,52 @@
+---
+
+# Ensure apt-transport-https and dirmngr are installed
+- name: Ensure apt-transport-https and dirmngr are installed
+  ansible.builtin.apt:
+    name: [apt-transport-https,dirmngr]
+    state: present
+  become: true
+
+
+# Add Scylla apt key
+- name: Add Scylla apt key
+  ansible.builtin.apt_key:
+    keyserver: hkp://keyserver.ubuntu.com:80
+    id: 5e08fbd8b5d6ec9c
```

Comment:
same variable thing ... this id can change from time to time, hardcode is not good

Reply:
Then you change the OS template and you are done. There's no guarantee about having the same keyserver and key for Ubuntu and Debian. If you think about it, this only depends of the OS itself.